mcstatus/status_response.py

```diff
@@ -1,57 +1,7 @@
-from __future__ import annotations
+from mcstatus.responses import *  # noqa: F403 # type: ignore # wildcard import
```

Comment:
Even though it's purely for a deprecated file, I would still prefer to see this fully written out rather than using a star import here, also removes 2 noqas.

Reply:
Done in 8e7fed46562727da2e70f46e23dd0dec2e81aee0.